runtime/xamarin/runtime.h

```diff
@@ -85,7 +85,7 @@ typedef struct __attribute__((packed)) {
 
 typedef struct __attribute__((packed)) {
 	const uint32_t *protocol_tokens; // an array of token references to managed interfaces that represent protocols
-	const Protocol **protocols; // the corresponding native protocols
+	const __unsafe_unretained Protocol **protocols; // the corresponding native protocols
```

Comment:
I am very uncertain if this is correct.

Reply:
I believe this is correct, but please add a comment explaining why and when this is required (and is possible include the error message).